vagen/envs/primitive_skill/maniskill/skill_wrapper.py

```
@@ -0,0 +1,269 @@
+        # import pdb; pdb.set_trace()
+        # while not done and not skill_done:
+
+        while not skill_done:
+            eef_state={
+                "robot0_eef_pos":common.unbatch(common.to_numpy(self.env.unwrapped.agent.tcp.pose.p)),
+                "robot0_eef_quat":common.unbatch(common.to_numpy(self.env.unwrapped.agent.tcp.pose.q)),
+            }
+            action_ll, skill_done, skill_success = self.primitive_skill.get_action(action, eef_state)
+            if self.use_abs_skill:
+                action_ll[:3]-=self.env.unwrapped.agent.robot.pose.p[0].cpu().numpy()
+            obs, reward, terminated, truncated, info = super().step(action_ll)
```

Ruff (0.15.9)

[warning] Unpacked variable `obs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)